README.md

```diff
@@ -81,7 +81,7 @@ Assisted digital satisfaction surveys:
 
 ## Technical documentation
 
-Frontend is a Ruby on Rails application that renders the citizen-facing part of formats stored in the Content Store. It looks up the passed-in slug in the Content Store.
+Frontend is a Ruby on Rails application and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html). It renders the citizen-facing part of formats stored in the Content Store. It looks up the passed-in slug in the Content Store.
 
 It also serves the homepage as a hard-coded route.
```

Comment:
This part can also be moved to the intro.

The "Calendar JSON" section below feels like something that would be better moved into its own document in the `docs/` folder, which we can link to from a "Further documentation" subsection that we've been adding to READMEs.

Finally, the `Additional information for calendars` sections onwards could also have the same treatment applied. (Sorry I can't comment on the actual lines - GitHub seems to have removed a feature!)

Reply:
👍